swatch-product-configuration/src/main/resources/subscription_configs/RHEL/RHEL_for_x86_EUS.yaml

```diff
@@ -0,0 +1,18 @@
+---
+platform: RHEL
+
+id: rhel-for-x86-eus
+
+#includedSubscriptions:
+#  - rhel-for-x86
+
+variants:
+  - tag: RHEL for x86 EUS
```

Comment:
We should use kebab-style for our tag names to make it more clear that this is more of an ID than a user readable string